__tests__/app.test.js

```diff
@@ -26,3 +27,23 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: Responds with an object of all the topics", () => {
```

Comment:
Why are the topics coming back on a key of "result"? That's not semantic and doesn't tell me what is stored on that key.